eng/pipelines/templates/jobs/build.yml

```diff
@@ -76,6 +76,23 @@ jobs:
     inputs:
       summaryFileLocation: $(CoverageFile)
 
+  - bash: |
```

Comment:
Ideally, this would all go into a local powershell script so devs can run it locally.

Reply:
good point!, moved to a PowerShell script.